app/views/admin/comments/index.html.erb

```diff
@@ -1,49 +1,48 @@
+          <div class="flex gap-2 items-center">
             <% if comment.user %>
-              <a href="<%= comment.user.path %>" target="_blank" rel="noopener">
-                <img class="rounded" height="30" src="<%= comment.user.profile_image_url_for(length: 50) %>" alt="<%= comment.user.username %> profile" loading="lazy" /> <%= comment.user.username %>
-              </a>
-            <% end %>
-            <% if comment.user && comment.user.twitter_username.present? %>
-              |
-              <a href="https://twitter.com/<%= comment.user.twitter_username %>" class="inline-flex items-center">
-                <%= crayons_icon_tag(:twitter, aria_hidden: true) %> @<%= comment.user.twitter_username %>
+              <a href="<%= comment.user.path %>" target="_blank" rel="noopener" class="shrink-0 c-link">
+                <img width="32" height="32" class="radius-full block" src="<%= comment.user.profile_image_url_for(length: 64) %>" alt="<%= comment.user.username %> profile" loading="lazy" />
               </a>
             <% end %>
+            <p>
+              <a href="<%= comment.user.path %>" target="_blank" rel="noopener" class="c-link c-link--branded fw-bold"><%= comment.user.username %></a>
+              on:
+              <a href="<%= comment.commentable.path %>" class="c-link c-link--branded"><%= comment.commentable.title %></a>
+            </p>
           </div>
-          <div class="mt-3 d-flex justify-content-end">
-            <a class="btn btn-primary" href="<%= comment.path %>" target="_blank" rel="noopener">View</a>
-            <% if comment.reactions.where(user_id: current_user.id).empty? %>
-              <%= form_tag(reactions_path, remote: true, class: "d-inline ml-2") do %>
-                <%= hidden_field_tag(:reactable_type, "Comment") %>
-                <%= hidden_field_tag(:reactable_id, comment.id) %>
-                <button class="btn btn-danger">❤️ Like</button>
-              <% end %>
-            <% end %>
-          </div>
-        </div>
+        </header>
+      <% end %>
+      <div class="text-styles text-styles--tertiary">
+        <%= sanitize comment.processed_html,
+                     tags: %w[strong em p h1 h2 h3 h4 h5 h6 i u b code pre br ul ol li small sup img a span hr blockquote],
+                     attributes: %w[href strong em ref rel src title alt class] %>
       </div>
-    </div>
-  </div>
-<% end %>
+      <footer class="fs-s flex gap-4 border-t-1 border-base-10 border-solid border-0 pt-2 -mx-2 pl-2 justify-between items-center">
+        <span>❤️ <%= pluralize(comment.public_reactions_count, "like") %></span>
+        <% if comment.reactions.where(user_id: current_user.id).empty? %>
+          <%= form_tag(reactions_path, remote: true) do %>
+            <%= hidden_field_tag(:reactable_type, "Comment") %>
+            <%= hidden_field_tag(:reactable_id, comment.id) %>
+            <button class="c-btn">❤️ Like</button>
```

Comment:
Is it weird that this now appears in the middle of the footer?

![like button in the middle of the footer](https://user-images.githubusercontent.com/20773163/152149838-5876aed7-b57e-451c-b0cc-7d9896eeec2b.png)

Reply:
yea, probably. ill fix it.